common/calibration-model.h

```diff
@@ -6,6 +6,7 @@
 #include "notifications.h"
 #include <rsutils/number/float3.h>
 
+#include "../src/ds/d500/d500-private.h"
```

Comment:
If this is only for mini_intrinsics it is better to forward declare mini_intrinsics and include in the cpp file.